src/collections/_documentation/workflow/discover2/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Discover V2'
+sidebar_order: 4
+---
+
+{% capture __alert_content -%}
+This version of Discover is in beta. Discover V1 will deprecate at the end of February 2020.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Our Discover tool combines the best parts of our Events and existing Discover features, providing visibility into your data across environments and building upon and enriching your error data. This new feature will allow you to query and unlock deeper insights into the health of your entire system and answer critical business questions --- all in one place.
+
+[{% asset discover/discover-homepage.png alt="Full view of Discover homepage with minigraphs and button to build new queries." %}]({% asset discover/discover-homepage.png @path %})
+
+## Discover Homepage
+
+The Discover homepage offers views of all your saved queries and Sentry's pre-built queries so you can quickly dive into areas that need immediate attention.
+
+### Minigraphs
```

Comment:
Is the word "Minigraphs" used anywhere in the UI?

Reply:
^ @bowencai8 @dashed

Reply:
No they're not. 

Re-reading over this paragraph, "Query Cards" might be a better terminology. I prefer Bowen to confirm.

Reply:
Query Cards sounds good to me as well.

Reply:
Done!